src/java.base/share/classes/javax/crypto/EncryptedPrivateKeyInfo.java

```diff
@@ -74,7 +74,8 @@ public class EncryptedPrivateKeyInfo {
      * its ASN.1 encoding.
      * @param encoded the ASN.1 encoding of this object. The contents of
      * the array are copied to protect against subsequent modification.
-     * @exception NullPointerException if the <code>encoded</code> is null.
+     * @exception NullPointerException if the <code>encoded</code> is
+     * <code>null</code>.
```

Comment:
Use `{@code }` instead of `<code></code>` for all of the javadoc?

Reply:
using `{@code }`